lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -235,7 +235,7 @@
             <div><?php print render($signup_data_form); ?></div>
             <?php if ($register_voters && dosomething_campaign_feature_on($campaign, 'social_share_unique_link')) : ?>
               <div class="modal__block">
-                <?php print t('Copy and paste your share link:') ?>
+                <?php print t('Use your unique registration link below for this competition. You must share this link to qualify for this competition.') ?>
```

Comment:
the custom social share language is currently being used as what the text to open the share modal says. The text before the link is also different depending on if it's in the competitions modal or the share modal, so unless we want to add more fields now I think it makes sense to use the share language field as it is?